benchmarks/benchmarks/stats.py

```diff
@@ -174,3 +178,51 @@ def time_binned_statistic_dd_reuse_bin(self, statistic):
         stats.binned_statistic_dd(
             [self.inp[0], self.inp[1]], self.inp[2], statistic=statistic,
             binned_statistic_result=self.ret)
+
+
+class ContinuousFitAnalyticalMLEOverride(Benchmark):
+    # list of distributions to time
+    dists = ["pareto", "laplace", "beta", "ncf"]
+    # add custom values for rvs and fit, if desired, for any distribution:
+    # key should match name in dists and value should be list of loc, scale,
+    # and shapes
+    custom_input = {"beta": [1, 1, 1, 1]}
+    fnames = ['floc', 'fscale', 'f0', 'f1', 'f2']
+    fixed = {}
+
+    param_names = ["distribution", "loc_fixed", "scale_fixed",
+                   "shape1_fixed", "shape2_fixed", "shape3_fixed"]
+    params = [dists, * [[True, False]] * 5]
+
+    def setup(self, dist_name, loc_fixed, scale_fixed, shape1_fixed,
+              shape2_fixed, shape3_fixed):
+        exec("self.distn= stats." + dist_name)
```

Comment:
```suggestion
        exec("self.distn = stats." + dist_name)
```
I'm ok with `exec`, but others might not like it. We'll see.

Reply:
I'll change it to `eval` instead of `exec` so it can be on the right side of an assignment. I'm not attached to using it like this, it just was the easiest way to allow distributions to be input as strings.

Using a string seems much preferable to passing in the distribution directly
![Screen Shot 2020-08-05 at 3 42 40 PM](https://user-images.githubusercontent.com/44255917/89471167-4d185b80-d732-11ea-8ffc-9b644ff70b7f.png)

Whereas it could otherwise just show the string.